data_diff/dbt_parser.py

```diff
@@ -0,0 +1,299 @@
+from collections import defaultdict
```

Comment:
to help me review: are there changes in addition to moving this class to a file? Or is it just moving the code?

Reply:
No, there are no changes, just moved the class to a separate file.